tests/test_material.py

```diff
@@ -98,16 +98,17 @@ def test_material_str(self):
         in_str = "M20 1001.80c 0.5 8016.80c 0.4 94239.80c 0.1"
         input_card = Input([in_str], BlockType.DATA)
         material = Material(input_card)
-        answers = """MATERIAL: 20 fractions: atom
- H-1   (80c) 0.5
- O-16  (80c) 0.4
-Pu-239 (80c) 0.1
+        answers = """\
+MATERIAL: 20 fractions: atom
+ZAID=1001, Z=1, A=1, element=hydrogen, library=80c 0.5
+ZAID=8016, Z=8, A=16, element=oxygen, library=80c 0.4
+ZAID=94239, Z=94, A=239, element=plutonium, library=80c 0.1
```

Comment:
I think keeping this is makes a worse representation. I think the old `repr` should just be removed and either the old `str` be kept, or a simpler form be used.

Reply:
This is the `__str__`. I'll change it to the old `__repr__` and update the material test accordingly.

Reply:
This still feels very verbose.

Reply:
See the new `Isotope.nuclide_str()` and let me know what you think

Reply:
kept old `__str__`